### PR TITLE
clean up home

### PR DESCRIPTION
**Pull request by johngruen**
This issue does a few things: 
- Replaces all screenshots icons on the home page with the current
- removes `no-warn` error from `sass-lint` (this rule triggers a warning on our responsive mixin)
- changes the globbing pattern for sass linting so that partials are linted, and corrects all linter errors and warnings we previously did not see
- moves homepage CSS to it's own file in the css directory with all other files
- lightly modifies home page styles to better handle variable widths
- corrects broken button states and images for the Firefox download button

### CODE REVIEW

**Comment by johngruen** [comment]
Note, filed #3310 as an eventual follow up

**Comment by johngruen** [comment]
Looks like this is doing that nightly fail thing :)

File: static/css/home.scss [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+    bottom: 0;
+    right: -10%;
+  }
+
+  section {
+    min-height: 500px;
+  }
+}
+
+@media (max-width: 768px) {
```
Comment (pdehaan) [inline static/css/home.scss]: Not sure if you want to use the `@include respond-to("small") {` style helpers for consistency.
🤷‍♀️

File: static/css/home.scss [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+#section-2 .section-image,
+#section-4 .section-content {
+  order: 1;
+}
+
+#section-1 .section-image,
+#section-2 .section-content,
+#section-4 .section-image {
+  order: 2;
+}
```
Comment (pdehaan) [inline static/css/home.scss]: Not sure if I'm missing something, but wouldn't this just overrule what you did on lines 316-320 above?

**Pull request merged**